website/src/pages/index.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import classnames from 'classnames';
+import Layout from '@theme/Layout';
+import Link from '@docusaurus/Link';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import withBaseUrl from '@docusaurus/withBaseUrl';
+import styles from './styles.module.css';
+
+const logoMain = [{
+    imageLogo: 'img/owl.png',
+}];
+
+
+const features = [
+    {
+        title: <>Maintainable</>,
+        imageUrl: 'img/tools.png',
+        description: (
+            <>
+                Write maintainable tests for your React Native apps
+            </>
+        ),
+    },
+    {
+        title: <>Reliable</>,
+        imageUrl: 'img/hit.png',
+        description: (
+            <>
+                Promotes testing public APIs and avoiding implementation details
+            </>
+        ),
+    },
+    {
+        title: <>Community Driven</>,
+        imageUrl: 'img/locomotive.png',
+        description: (
+            <>
+                Supported by React Native community and its core contributors
+            </>
+        ),
+    },
+];
+
+
+function Home() {
+    const context = useDocusaurusContext();
+    const {siteConfig = {}} = context;
+    return (
+            <Layout
```

Comment:
This indentation is a bit off. Can you please run prettier through all of the JS inside website/ dir?

Reply:
Got it